examples/flags/README.md

```diff
@@ -68,3 +68,37 @@ To get additional verbose logs
 ```shell
 ./flags.test --assess es --v 2
 ```
+
+You can also pass additional custom flags
+
+```shell
+./flags.test --myflag myvalue
+```
+
```

Comment:
Would you mind turning this code into an actual go file that can be run or update the existing flag test itself to include this behavior or customizing the flags? That would be really cool to have a working example that you can run as well instead of just read.

Reply:
Same suggestion. Or it may even makes more sense to create a new example under `examples/custom-flags/`, with README.md and a go example to show how to do it.

Reply:
Done, does this look better?